UI fixes

Rachel-AG opened the pull request:
- Fix description overflow in dashboard
- Replace courses placeholder image
- Replace website logo
- Fix modal colors
- Improve chat UI

ryanguai commented:
What were the changes made to the chat UI? I moved the chat from the modal in TemplateRow to a new page, so any change made in the modal won't be reflected. But I also made some UI improvements in the new page I made

Rachel-AG commented:
> What were the changes made to the chat UI? I moved the chat from the modal in TemplateRow to a new page, so any change made in the modal won't be reflected. But I also made some UI improvements in the new page I made

I removed the document list as discussed and change the color of the toggle since it was not visible when the modal is white. If you re-use the toggle in the new page, it'll be great if you can use the following tag for the toggles:
   
```
 <span className='text-gray-900 dark:text-white mr-3'>
          Retrieval Augmented Generation
 </span>
```

ryanguai commented:
I changed it to a dropdown since we have many options now but we can discuss again what UI is best
